src/app/layout/search/search.component.scss

```diff
@@ -86,13 +96,18 @@
   :host {
     .search-box {
       padding: 12px 0px;
+      justify-content: center;
+      max-width: -webkit-calc(100% - 28px);
+      max-width: -moz-calc(100% - 28px);
```

Comment:
Seems like vendor prefix is not required since the cal is not an experimental feature anymore

Reply:
Ok, ok